types/package.json

```diff
@@ -6,6 +6,7 @@
   "repository": "https://github.com/desmos-labs/contracts-utils",
   "license": "Apache-2.0",
   "scripts": {
+    "pull-cw721-schemas": "ts-node src/pull-cw721-schemas.ts",
```

Comment:
Please move this inside `pull-schemas` so that an user just needs to run `yarn pull-schemas` to download all the required schemas.